test/e2e_node/runner/remote/run_remote.go

```diff
@@ -218,35 +215,36 @@ func main() {
 	gceImages := &internalImageConfig{
 		images: make(map[string]internalGCEImage),
 	}
+	// Parse images from given config file and convert them to internalGCEImage.
 	if *imageConfigFile != "" {
 		configPath := *imageConfigFile
 		if *imageConfigDir != "" {
 			configPath = filepath.Join(*imageConfigDir, *imageConfigFile)
 		}
 
-		// parse images
 		imageConfigData, err := ioutil.ReadFile(configPath)
 		if err != nil {
 			klog.Fatalf("Could not read image config file provided: %v", err)
 		}
+		// Unmarshal given image config file into a map shortName->GCEImage, e.g cos-stable->cos-stable-81-12871-103-0.
```

Comment:
nit:
```suggestion
		// Unmarshal then given image config file. All images for this test run will be organized in a map
		// where each GCEImage value will be associated with the image's name as a key.
```

or something like that, just be more explicit about what is going on.